addons/base_automation/models/ir_actions_server.py

```diff
@@ -10,13 +10,18 @@
 class IrActionsServer(models.Model):
     _inherit = "ir.actions.server"
 
-    name = fields.Char(compute='_compute_name', store=True, readonly=False)
-
     usage = fields.Selection(selection_add=[
         ('base_automation', 'Automation Rule')
     ], ondelete={'base_automation': 'cascade'})
     base_automation_id = fields.Many2one('base.automation', string='Automation Rule', ondelete='cascade')
 
+    def _get_linkable_childs_domain_parts(self):
+        # As automation rules' actions does not have a parent,
+        # we make sure multi actions can not link to automation rules' actions.
+        return super()._get_linkable_childs_domain_parts() + [
```

Comment:
you don't really control the call to super, so use expression.py to write `expression.AND( super, newdomain)`

Reply:
see my comment in the base module